irods_testing_environment/irods_setup.py

```diff
@@ -421,17 +421,17 @@ def setup_irods_catalog_consumers(ctx,
     rc = 0
 
     with concurrent.futures.ThreadPoolExecutor() as executor:
-        futures_to_containers = {
+        futures_to_csc_instances = {
```

Comment:
csc always makes me confused.    consider 'consumer', but yes, i know i know...

Reply:
I just wanted to make sure the variable name better reflected what it was - these are not Containers, but they are instance numbers.